experimental/GITFans/GITFans.jl

```diff
@@ -112,22 +112,22 @@ such that the matrix `S` has full rank and such that `I` is
 monomial-free (see [`is_monomial_free`](@ref)) w.r.t. the variables `x_i`
 for which the `i`-th row of `Q` is not contained in `S`.
 """
-function orbit_cones(I::Oscar.MPolyIdeal, Q::Matrix{Int}, G::PermGroup = symmetric_group(1))
+function orbit_cones(I::MPolyIdeal, Q::Matrix{Int}, G::PermGroup = symmetric_group(1))
     nr_variables, projected_dimension = size(Q)
 
     collector_cones = []
 
     # We need not consider sets of smaller size because of the rank condition.
     for k in projected_dimension:nr_variables
-        for orb in orbits(gset(G, on_sets, Hecke.subsets(nr_variables, k)))
+        for orb in orbits(gset(G, on_sets, subsets(nr_variables, k)))
             i = representative(orb)
             current_mat = Q[i,:]
             if rank(current_mat) == projected_dimension &&
                is_monomial_free(I, setdiff(1:nr_variables, i))
-                cone = Polymake.polytope.Cone(INPUT_RAYS = current_mat)
-                cone.FACETS
-                cone.RAYS
-                if ! any(j -> Polymake.polytope.equal_polyhedra(j, cone),
+                cone = positive_hull(current_mat)
+                facets(cone)
+                rays(cone)
+                if ! any(j -> j == cone,
                       collector_cones)
```

Comment:
Maybe this would also work now in the much nicer new setup? It would arguably easier to understand, I think
```suggestion
                if !(cone in collector_cones)
```